docs/rules/google_project_iam_member_invalid_member_format.md

```diff
@@ -0,0 +1,30 @@
+# google_project_iam_member_invalid_member_format
+
+Check iam member format.
+
+## Example
+
+```hcl
+resource "google_project_iam_member" "iam_member" {
+	member = "jane@example.com"
+}
+```
+
+```
+$ tflint
+1 issue(s) found:
+
+Error: jane@example.com is an invalid member format. (google_project_iam_member)
+
+  on template.tf line 1:
+   1: resource "google_project_iam_member" "iam_member" {
+
+```
+
+## Why
+
+A member value you wrote is invalid. `terraform apply` will result in failure.
```

Comment:
It is better to mention why errors are more likely to occur. For example, it might be helpful to add an explanation like the following:

> In particular, don't forget to add a prefix to the value. Since the principal type is identified with a prefix, a value without a prefix cannot be interpreted correctly.

In addition, I recommend that you add external references in the doc.
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_iam#member/members
https://cloud.google.com/iam/docs/overview#concepts_related_identity
https://cloud.google.com/iam/docs/overview#cloud-iam-policy

Reply:
It may be best to list valid values as follows:

- `allUsers`
- `allAuthenticatedUsers`
- `user:{emailid}`
- `serviceAccount:{emailid}`
- `group:{emailid}`
- `domain:{domain}`

Reply:
Added more explanation https://github.com/terraform-linters/tflint-ruleset-google/pull/121/commits/c2334c1d4f82814ad566529aa13d0cebe1d1f9eb